charts/alfresco-ooi-service/values.yaml

```diff
@@ -0,0 +1,87 @@
+repository:
+  edition: "Enterprise"
+global:
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+ooiService:
```

Comment:
This level of values doesn't make sense here, otherwise you'll have an additionnal level incase of usage from  dependecies: `.Values.dependency_name.ooiService.*`, which we don't want